vts/trustedservices/trustedservices_grpc.go

```diff
@@ -418,17 +424,22 @@ func (c *GRPC) initEvidenceContext(
 	return appraisal, err
 }
 
-func (c *GRPC) getTrustAnchor(id string) (string, error) {
-	values, err := c.TaStore.Get(id)
-	if err != nil {
-		return "", err
-	}
+func (c *GRPC) getTrustAnchors(id []string) ([]string, error) {
+
+	ta_values := make([]string, 0)
```

Comment:
golang wants us to use camelCase:

```suggestion
	taValues := make([]string, 0)
```